content/ja/docs/concepts/security/pod-security-admission.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Podのセキュリティアドミッション
+content_type: concept
+weight: 20
+min-kubernetes-server-version: v1.22
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+Kubernetesの[Podセキュリティの標準](/ja/docs/concepts/security/pod-security-standards/)はPodに対して異なる分離レベルを定義します。
+これらの標準によって、Podの動作をどのように制限したいかを、明確かつ一貫した方法で定義することができます。
+
+ベータ版機能として、Kubernetesは[PodSecurityPolicies](/docs/concepts/policy/pod-security-policy/)の後継である組み込みの _Pod Security_ {{< glossary_tooltip text="アドミッションコントローラー" term_id="admission-controller" >}}を提供しています。
```

Comment:
```suggestion
ベータ版機能として、Kubernetesは[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)の後継である組み込みの _Pod Security_ {{< glossary_tooltip text="アドミッションコントローラー" term_id="admission-controller" >}}を提供しています。
```

(which avoids a Japanese reader having to think about English pluralization rules)